README.md

```diff
@@ -82,20 +82,21 @@ Each request is verified using a passcode generated at deploy time. Passcodes re
 
 ### Token
 
-This endpoint returns a Programmable Video Access token. When `create_room` is true, it will create a room and [Twilio conversation](https://www.twilio.com/docs/conversations/api/conversation-resource) associated with the room. This token is used by the above mentioned Video Apps to connect to a video room and a conversation.
+This endpoint returns a Programmable Video Access token. When `create_room` is true, it will create a room, and when `create_conversation` is true, it will create a [Twilio conversation](https://www.twilio.com/docs/conversations/api/conversation-resource) associated with the room. This token is used by the above mentioned Video Apps to connect to a video room and a conversation.
 
 ```shell
 POST /token
 ```
 
 #### Parameters
 
-| Name            | Type      | Description                                                                            |
-| --------------- | --------- | -------------------------------------------------------------------------------------- |
-| `passcode`      | `string`  | **Required**. The application passcode.                                                |
-| `user_identity` | `string`  | **Required**. The user's identity.                                                     |
-| `room_name`     | `string`  | **Required when `create_room` is `true`** A room name that will be used to create a token scoped to connecting to only one room. |
-| `create_room`   | `boolean` | (default: `true`) When false, a room will not be created when a token is requested.    |
+| Name                  | Type      | Description                                                                            |
+| --------------------- | --------- | -------------------------------------------------------------------------------------- |
+| `passcode`            | `string`  | **Required**. The application passcode.                                                |
+| `user_identity`       | `string`  | **Required**. The user's identity.                                                     |
+| `room_name`           | `string`  | **Required when `create_room` is `true`** A room name that will be used to create a token scoped to connecting to only one room. |
+| `create_room`         | `boolean` | (default: `true`) When false, a room will not be created when a token is requested.    |
+| `create_conversation` | `boolean` | (default: `false`) When true, a Twilio Conversation will be created (if it doesn't already exist) and a participant will be added to it when a token is requested. `create_room` must also be `true`.   |
```

Comment:
We should link Twilio Conversation here

Reply:
Here I'll link to the actual Conversation resource since it's what I'm referring to: https://www.twilio.com/docs/conversations/api/conversation-resource